boards/nucleo144-f746/Makefile.features

```diff
@@ -0,0 +1,12 @@
+# Put defined MCU peripherals here (in alphabetical order)
+FEATURES_PROVIDED += periph_cpuid
+FEATURES_PROVIDED += periph_gpio
+FEATURES_PROVIDED += periph_hwrng
+FEATURES_PROVIDED += periph_timer
```

Comment:
For completeness, we could add some PWM configuration. Maybe it's better to add it in a follow-up PR ?